<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rcalibrationClass</class>
 <widget class="QDialog" name="rcalibrationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>209</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_ReticleCalibration">
     <property name="title">
      <string>Reticle calibraion (you can drag reticle by its center)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="margin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="l_1">
        <property name="text">
         <string>Square size</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_SquareSize"/>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QCheckBox" name="checkBox_TwoAxis">
        <property name="text">
         <string>two axis</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="pushButton_FindStar">
        <property name="text">
         <string>Find star</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBox_AutoMode">
        <property name="text">
         <string>auto mode</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBox_DriftTime">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="value">
         <number>25</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="3">
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="l_2">
        <property name="text">
         <string>recticle-X</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="l_3">
        <property name="text">
         <string>reticle-Y</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="l_4">
        <property name="text">
         <string>recticle-Angle</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3" colspan="2">
       <widget class="QLabel" name="l_RStatus">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>State:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QDoubleSpinBox" name="spinBox_ReticleX"/>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="spinBox_ReticleY"/>
      </item>
      <item row="3" column="2">
       <widget class="QDoubleSpinBox" name="spinBox_ReticleAngle">
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QPushButton" name="pushButton_StartCalibration">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_VideoCalibration">
     <property name="title">
      <string>Accumulate dark frames</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="margin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="l_5">
        <property name="text">
         <string>Frame count</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="l_VStatus">
        <property name="text">
         <string>State:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QSpinBox" name="spinBox_FrameCount">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_StartVideoCalibration">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBox_SquareSize</tabstop>
  <tabstop>checkBox_TwoAxis</tabstop>
  <tabstop>pushButton_FindStar</tabstop>
  <tabstop>checkBox_AutoMode</tabstop>
  <tabstop>spinBox_DriftTime</tabstop>
  <tabstop>spinBox_ReticleX</tabstop>
  <tabstop>spinBox_ReticleY</tabstop>
  <tabstop>spinBox_ReticleAngle</tabstop>
  <tabstop>pushButton_StartCalibration</tabstop>
  <tabstop>spinBox_FrameCount</tabstop>
  <tabstop>pushButton_StartVideoCalibration</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
